Content.Server/Explosion/EntitySystems/ExplosionSystem.Processing.cs

```diff
@@ -514,6 +514,8 @@ public void DamageFloorTile(TileRef tileRef,
     {
         if (_tileDefinitionManager[tileRef.Tile.TypeId] is not ContentTileDefinition tileDef)
             return;
+        if(tileDef.Indestructible)
```

Comment:
Please combine the new check with the existing one, since the outcome is the same

        if (_tileDefinitionManager[tileRef.Tile.TypeId] is not ContentTileDefinition tileDef
            || tileDef.Indestructible)
            return;